app/Http/Controllers/Table/FdbTablesController.php

```diff
@@ -152,6 +160,8 @@ public function formatItem($fdb_entry)
             'vlan' => $fdb_entry->vlan ? $fdb_entry->vlan->vlan_vlan : '',
             'description' => '',
             'dnsname' => $ip_info['dns'],
+            'first_seen' => strval($fdb_entry->created_at),
```

Comment:
created_at should be an instance of carbon, so you can use that to get the format you want.

Reply:
Done. Last seen is now in diffForHumans()